src/OpenSage.Game/Client/Drawable.cs

```diff
@@ -403,7 +403,7 @@ internal void Destroy()
 
         public void Persist(StatePersister reader)
         {
-            reader.PersistVersion(5);
+            var version = reader.PersistVersion(7);
```

Comment:
I think we need to persist this don't we? That way when we save, we save the correct version? Or is the plan to save everything with the latest version?

Reply:
I was thinking we'd save everything with the latest version, yes. The only reason to save with older versions would be "export" a save file that's compatible with original versions of older games, and I don't think there's really a need for that. In fact I mainly see importing .sav files as a development aid for us, to help us learn about the original games.

I _did_ save the version in the W3D parser because I thought maybe we could use it as the basis of a .w3d converter between SAGE games, but I don't see the same value for save files.

What do you think?

Reply:
I'm totally fine with saving everything as the latest version - but is that always possible? For example, there are cases where going from Generals to Zero Hour bytes are _removed_, which seems like something that could get us if maybe in Generals certain data was needed for a given behavior but in Zero Hour that behavior was removed from the engine and thus no longer supported.

I agree that we don't need to worry about saving a game in OpenSAGE and then loading it in vanilla Generals - I think the level of engineering effort required for something like that would be astronomical compared to the value gained.